docs/writing-rules/experiments/join-mode/overview.md

```diff
@@ -196,7 +196,7 @@ Short for references, `refs` is a list of external rules that make up your code
 
 ### `rule`
```

Comment:
@armchairlinguist @minusworld should this actually be `rules`? I don't see `rule:` key in the examples on this page.